src/components/NavBar.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
```

Comment:
import hooks only from react, you don't need the React
```suggestion
import {useState, ...} from "react";
```

Reply:
Sure, I will do that.